network-api/networkapi/wagtailpages/pagemodels/customblocks/video_block.py

```diff
@@ -16,6 +18,31 @@ class VideoBlock(blocks.StructBlock):
         required=False,
         help_text='Optional URL for caption to link to.'
     )
+    wide_video = blocks.BooleanBlock(
+        required=False,
+        default=False,
+        help_text='Would you like to use a wider video on desktop?'
+    )
+    full_width_video = blocks.BooleanBlock(
+        required=False,
+        default=False,
+        help_text='Would you like to use a full width video?',
+    )
```

Comment:
Sabrina's idea for a radio block might be a better idea than two checkboxes. I've left a comment with links (above) to see example radio button code. It'll help provide a better UX for editors, and we could probably remove the validation if we're using a radio block.